lib/build.js

```diff
@@ -174,15 +174,20 @@ function linkBins (pkg, folder, parent, gtop, cb) {
   log.verbose('linkBins', [pkg.bin, binRoot, gtop])
 
   asyncMap(Object.keys(pkg.bin), function (b, cb) {
+    /**
```

Comment:
We do it because on Windows `npm install` creates inside `node_modules` files `package.json` with  `bin` section that contains records with non-POSIX path separators, like:

```
  "bin": {
    "sshpk-conv": "bin\\sshpk-conv",
    "sshpk-sign": "bin\\sshpk-sign",
    "sshpk-verify": "bin\\sshpk-verify"
  },
```

And then `npm rebuild` on Linux machine failed resolve path, like:

```
/bundle/bundle/programs/server/npm/node_modules/meteor/npm-container/node_modules/sshpk/bin\\sshpk-conv
```